tests/test_strategy_state_visualisation_multipair.py

```diff
@@ -386,3 +393,60 @@ def test_visualise_strategy_state(
         open_plotly_figure_in_browser(image, height=2000, width=1000)
         open_plotly_figure_in_browser(image_no_detached, height=2000, width=1000)
         open_plotly_figure_in_browser(image_no_indicators, height=2000, width=1000)
+
+
+def test_visualise_strategy_state_overriden_pairs(
+    logger: logging.Logger,
+    strategy_universe,
+):
+    """Visualise strategy state as a bunch inline images."""
+
+    routing_model = generate_simple_routing_model(strategy_universe)
+
+    # Run the test
+    state, strategy_universe, debug_dump = run_backtest_inline(
+        start_at=START_AT,
+        end_at=END_AT,
+        client=None,  # None of downloads needed, because we are using synthetic data
+        cycle_duration=TRADING_STRATEGY_CYCLE,
+        decide_trades=decide_trades,
+        create_trading_universe=None,
+        universe=strategy_universe,
+        initial_deposit=INITIAL_DEPOSIT,
+        reserve_currency=ReserveCurrency.busd,
+        trade_routing=TradeRouting.user_supplied_routing_model,
+        routing_model=routing_model,
+        log_level=logging.WARNING,
+        allow_missing_fees=True,
+    )
+
+    # visualise only one pair
+    state.visualisation.set_visualised_pairs([
+        strategy_universe.data_universe.pairs.get_pair_by_id(99),
```

Comment:
You can use `strategy_universe.get_pair_by_human_description` and it returns `TradingPairIdentifier` directly

Reply:
`get_pair_by_human_description()` actually returns `DEXPair` as well 😂, but I got passed it with `get_trading_pair()`